wandi/easyadmin-plus-bundle/0.6/config/packages/wandi_easyadmin_plus.yaml

```diff
@@ -0,0 +1,60 @@
+#wandi_easy_admin_plus:
```

Comment:
We should not have config in comments like this. Either remove it or uncomment it.